modules/ocf/manifests/packages/docker.pp

```diff
@@ -44,35 +40,36 @@
     ocf::systemd::override { 'set-docker-socket-group':
       unit    => 'docker.socket',
       content => "[Socket]\nSocketGroup=${admin_group}\n",
-      require => Package['docker-engine'],
+      require => Package['docker-ce'],
       notify  => Exec['docker-socket-update'];
     }
   }
 
   if $autoclean {
     cron {
       'clean-old-docker-containers':
-        # days is intentionally plural
-        command => "docker ps -a --filter status=exited | grep -E 'Exited \\([0-9]+\\) [0-9]+ (days|weeks?|months?|years?) ago' | awk '{print \$1}' | chronic xargs -r docker rm",
+        command => "chronic docker container prune --filter until=${container_max_age} -f",
         hour    => 1,
         minute  => 3;
 
-      'clean-old-created-docker-containers':
-        # days is intentionally plural
-        command => "docker ps -a --filter status=created | grep -E '(days|weeks?|months?|years?) ago' | awk '{print \$1}' | chronic xargs -r docker rm",
-        hour    => 1,
-        minute  => 5;
```

Comment:
Using `docker container prune` already deletes containers with status `created`, so this is deleted.